Makefile

```diff
@@ -198,46 +372,77 @@ else
 		${KIND_CLUSTER}-control-plane bash
 endif
 
+# Run a shell in docker image, as root
+kind-shell-root:
+	docker exec -it -e KUBECONFIG=/etc/kubernetes/admin.conf \
+		-w ${GOPATH}/src/github.com/istio-ecosystem/istio-installer \
+		${KIND_CLUSTER}-control-plane bash
+
+
 # Grab kind logs to $GOPATH/out/logs
 kind-logs:
 	mkdir -p ${OUT}/logs
 	kind export logs --name  ${KIND_CLUSTER} ${OUT}/logs
 
 # Build the Kind+build tools image that will be useed in CI/CD or local testing
 # This replaces the istio-builder.
-docker.istio-builder: test/docker/Dockerfile ${GOPATH}/bin/kind ${GOPATH}/bin/helm ${GOPATH}/bin/go-junit-report ${GOPATH}/bin/repo
+docker.istio-builder: test/docker/Dockerfile ${GOPATH}/bin/istioctl ${GOPATH}/bin/ci2gubernator ${GOPATH}/bin/go-junit-report ${GOPATH}/bin/kind ${GOPATH}/bin/helm ${GOPATH}/bin/go-junit-report ${GOPATH}/bin/repo
 	mkdir -p ${GOPATH}/out/istio-builder
+	curl -Lo - https://github.com/istio/istio/releases/download/${STABLE_TAG}/istio-${STABLE_TAG}-linux.tar.gz | \
+    		(cd ${GOPATH}/out/istio-builder;  tar --strip-components=1 -xzf - )
 	cp $^ ${GOPATH}/out/istio-builder/
 	docker build -t istionightly/kind ${GOPATH}/out/istio-builder
 
+docker.buildkite: test/buildkite/Dockerfile ${GOPATH}/bin/kind ${GOPATH}/bin/helm ${GOPATH}/bin/go-junit-report ${GOPATH}/bin/repo
+	mkdir -p ${GOPATH}/out/istio-buildkite
+	cp $^ ${GOPATH}/out/istio-buildkite
+	docker build -t istionightly/buildkite ${GOPATH}/out/istio-buildkite
+
 # Build or get the dependencies.
-dep: ${GOPATH}/bin/kind ${GOPATH}/bin/helm
+dep:
+
+#${GOPATH}/bin/kind ${GOPATH}/bin/helm
+
+GITBASE ?= "https://github.com"
```

Comment:
these  two lines don't look right - shouldn't they be tabbed for Makefile to  accept the  values?

Reply:
Not for variables.